src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -206,6 +206,7 @@ export type Project = {
   latestDeploys: Record<string, Pick<Deploy, 'dateFinished' | 'version'>> | null;
   builtinSymbolSources?: string[];
   stats?: Array<[number, number]>;
+  latestRelease?: {version: string};
```

Comment:
I've come across a similar proptype in a few other places. I wonder if it is worth adding a named type for this?